### ISSUE TITLE
Unicode escaping seems not to work correctly

### ISSUE THREAD

**Issue by tmssngr** [vlang/v#6317]
**V version:** 2cb711e
**OS:** Win10

**What did you do?**
Ran following code:
```
fn main() {
	println('äöü\u0020\u00c4\u00d6\u00dc')
}
```

**What did you expect to see?**
```
äöü ÄÖÜ
``` 
**What did you see instead?**
```
äöü ���
```

**Comment by tpanj** [comment]
Same on Linux

**Comment by zakuro9715** [comment]
Unicode escaping seems not implemented in V. If it works, it handled by C compiler.

```
==================
                                             ^
/tmp/v/main.8618474909997106254.tmp.c:8645:23: error: character ' ' cannot be specified by a universal character name
        println(_SLIT("äöü\u0020\u00c4\u00d6\u00dc"));
                          ^~~~~~
/tmp/v/main.8618474909997106254.tmp.c:357:63: note: expanded from macro '_SLIT'
#define _SLIT(s) ((string){.str=(byteptr)("" s), .len=(sizeof(s)-1), .is_lit=1})
                                                              ^
2 errors generated.
...
==================
(Use `v -cg` to print the entire error message)

builder error:

```

**Comment by zakuro9715** [comment]
I'll fix this.

**Comment by zakuro9715** [comment]
Most of c compiler treat `\uXXXX` as utf8, but universal character is not necessarily utf8 In C standard. 

In some cases, `\u00c4` will be `0xc4`. https://github.com/vlang/v/runs/2093700018#step:11:15

I think, V should convert `\uXXXX` into utf8 bytes and generate `\xXX` sequence (e.g `'\u00c4'` -> `\xc3\x84'`)

@medvednikov what do you think?

**Comment by zakuro9715** [comment]
Not yet fixed

**Comment by zakuro9715** [comment]
fixed in #9298
